sonic-slave/Dockerfile

```diff
@@ -116,6 +116,7 @@ RUN apt-get update && apt-get install -y \
         libdb-dev \
         iptables-dev \
         swig \
+        ctags \
```

Comment:
Why adding ctags?

Reply:
On line L106, these packages are for mellanox sdk build. ctags is build dependency for python-sdk-api package

Reply:
Everyone has no access to MLNX_SDK_SOURCE, but need one extra package in sonic-slave. It is weird for a package to depend on ctags during build time. Could you check if you can remove this dependency internally?

---
In reply to: [404943374](https://github.com/Azure/sonic-buildimage/pull/4378#discussion_r404943374) [](ancestors = 404943374)